survey-runner-queue/ebs.tf

```diff
@@ -0,0 +1,9 @@
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
+  EC2_INSTANCE_TAG = "${var.env}-RabbitMQ"
```

Comment:
Someone who has access to pre-prod would have to do this as I do not have access to pre-prod